docs/Quality.md

```diff
@@ -94,13 +94,13 @@ Good details in a bug report include:
   * Trivial (eg. a copy error)
 
 ### Agile Manual QA Overflow
-Developing automated unit or UI tests are time consuming. It's almost impossible to maintain automation tests while a feature is being developed on at the same time. And often having enough time to develop unit tests and UI automation tests while developing is a luxury we don't get. So, to continue to maintain high quality, manual QA becomes necessary until the feature settles down.
+Developing automated unit or UI tests are time consuming. It's almost impossible to maintain automation tests while a feature is being developed. And often, having enough time to create tests while developing is a luxury we don't get. So, to continue to maintain high quality, manual QA becomes necessary until the feature settles down.
 
-To minimize human error, but maximize time to release, the best approach is to develop a Manual QA Document, or checklist, that captures items that should be tested before a release. These items should be both the normal vectors, and also the corner case vectors of the application. This document should also be updated regularly as more corner cases appear, bugs are found and squashed, and features are changed.
+To minimize human error, but maximize time to release, the best approach is to develop a Manual QA Document, or checklist, that captures items that should be tested before a release. These items should include both the normal vectors, and also corner case vectors of the application. This document should be updated regularly as more corner cases appear, bugs are found and squashed, and features are changed.
 
-Once there is time, and the churn on a feature has stopped, the items from the manual QA document should be converted to either unit tests, or UI automation tests. This creates an *overflow* type of process, like the coolant overflow tank in a car. As you develop and things are hot, new tests should "overflow" to the manual QA document, and then once development has cooled off, items should be removed from the manual QA document and converted to Unit or UI Automation tests.
+Once there is time, and churn on a feature ends, the items from the manual QA document should be converted to either unit tests, or UI automation tests. This creates an *overflow* process, like the coolant overflow tank in a car. As you develop, and things are hot, new tests should "overflow" to the manual QA document. Then, once development has cooled off, items should be removed from the manual QA document and converted to Unit or UI Automation tests.
```

Comment:
this sounds intriguing. i think following a BDD pattern for these manual tests could make it easier to convert them into acceptance tests. i'll submit a PR for short explanations of a few main testing strategies, which includes BDD.